DEVELOPMENT.md

```diff
@@ -0,0 +1,78 @@
+# Developement
+
+## Feature Release Flow
+
+### Branching Strategy
+
+- The `master` branch is intended to only contain features for the immediately upcoming release, whether it is a Height Coordinated Upgrade or a Spork
+- During development, all features should live on a feature branch.
+- For small features, this will be simple working branch, i.e. `kan/1-fix-known-issue`
+- For larger features, this may be a shared feature branch with other team members.
```

Comment:
Maybe mention the feature branch naming scheme here 
```suggestion
- For larger features, this may be a shared feature branch with other team members. Feature branches have the naming scheme `feature/<name>`.
```